src/connectors/refinement-list/connectRefinementList.js

```diff
@@ -75,6 +73,9 @@ export default function connectRefinementList(renderFn) {
 
     checkUsage({attributeName, operator, usage});
 
+    const formatItems = ({name: label, ...item}) =>
+      ({...item, label, value: label, highlighted: label});
```

Comment:
It's used here https://github.com/algolia/instantsearch.js/blob/master/src/widgets/refinement-list/defaultTemplates.js#L8 I didn't change this part should be the same as before.